lib/bundler_api/versions_file.rb

```diff
@@ -0,0 +1,81 @@
+require 'bundler_api'
+require 'bundler_api/cdn'
+
+class BundlerApi::VersionsFile
+  PATH = "versions.list"
+  def initialize(connection)
+    @conn = connection
+  end
+
+  def create_or_update
+    if File.exists? PATH
+      update
+    else
+      create
+    end
+  end
+
+  def create
+    content = Time.now.to_i.to_s
+    content += "\n---\n"
+    content += gems_for_new_file
+
+    File.open(PATH, 'w') do |io|
+      io.write content
+    end
+
+    BundlerApi::Cdn.purge_versions_list
```

Comment:
Since this is run by a rake task, and then the file will be checked in to git, I don't think it makes sense to purge the CDN when we run the rake task locally.